### ISSUE TITLE
UIActivityViewController , After saving the GIF, the photo opens as a still image

### ISSUE THREAD

**Issue by XavierLost** [SDWebImage/SDWebImage#2865]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

UIActivityViewController , After saving the GIF, the photo opens as a still image

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
UIActivityViewController does not support GIF. This is Apple's issue. Try to report radar to Apple: https://feedbackassistant.apple.com/

**Comment by dreampiggy** [comment]
See more here: https://stackoverflow.com/questions/35567969/how-can-i-share-a-gif-image-using-uiactivityviewcontroller-with-objective-c

It seems that you can use this API to share a GIF data (using SDWebImage's encoding API or manager callback with data) : https://developer.apple.com/documentation/uikit/uiactivityitemsource/1620456-activityviewcontroller?language=objc with `kUTTypeGIF`

**Comment by XavierLost** [comment]
thank you!
